backend/app/services/mcp_providers/core/config.py

```
@@ -0,0 +1,152 @@
+    ServerMappingConfig,
+)
+
+# Aliyun Bailian MCP Provider
+BAILIAN_CONFIG = MCPProviderConfig(
+    key="bailian",
+    name="阿里云百炼",
+    name_en="Aliyun Bailian",
+    description="阿里云大模型服务平台百炼 MCP 市场",
+    discover_url="https://bailian.console.aliyun.com/?tab=mcp#/mcp-market",
+    api_key_url="https://bailian.console.aliyun.com/?tab=app#/api-key",
+    token_field="bailian",
```

Ruff (0.15.2)

[error] Possible hardcoded password assigned to argument: "token_field"

(S106)